### PR TITLE
_redeemCollateralFromTrove

### PR DESCRIPTION
**Pull request by MaayanShore**

### CODE REVIEW

File: env/lib/python3.10/site-packages/nile/base_project/tests/__pycache__/test_contract.cpython-310.pyc
Comment (YehudisSternberg) [inline env/lib/python3.10/site-packages/nile/base_project/tests/__pycache__/test_contract.cpython-310.pyc:1]: מה זה?

**Review by YehudisSternberg — CHANGES_REQUESTED** [review]

**Comment by MaayanShore** [comment]
it is the _redeemCollateralFromTrove function part of the trove manager is there a problem?

**Comment by YehudisSternberg** [comment]
> it is the _redeemCollateralFromTrove function part of the trove manager is there a problem?

no but there are two more files added 
what are they?

**Comment by MaayanShore** [comment]
I changed a little thing in a different file and I saw only tow files changed

**Comment by YehudisSternberg** [comment]
> I changed a little thing in a different file and I saw only tow files changed

image

please do pr are with only one file changed

**Pull request closed without merging**